.github/workflows/ci.yml

```diff
@@ -279,37 +279,6 @@ jobs:
             --region="${{ env.AUTOPUSH_REGION }}" \
             --image="${{ env.DOCKER_REPO }}/github-metrics-aggregator:${{ env.DOCKER_TAG }}-amd64"
 
-  deployment_artifact_pipeline_autopush:
```

Comment:
By removing this, do you intend to pause the batch pipeline until after the cloud run job is up and running?

Reply:
Yes, I'll pause the regular dataflow pipeline once I'm ready to deploy the CR job version.